.gitignore

```diff
@@ -0,0 +1 @@
+build
```

Comment:
Thinking about it, I'd probably add more of the common ignorable items now while we're at it…
```suggestion
/build

# Mac
.DS_Store

#VSCode
.vscode

# Intellij
.idea/
*.iml
*.iws
```